subworkflows/nf-core/vcf_gather_bcftools/main.nf

```diff
@@ -0,0 +1,51 @@
+include { BCFTOOLS_CONCAT    } from '../../../modules/nf-core/bcftools/concat/main'
+include { BCFTOOLS_SORT      } from '../../../modules/nf-core/bcftools/sort/main'
+include { TABIX_TABIX        } from '../../../modules/nf-core/tabix/tabix/main'
+
+workflow VCF_GATHER_BCFTOOLS {
+
+    take:
+    ch_vcfs             // channel: [ meta, vcf, tbi ]
+    ch_scatter_output   // channel: [ meta, bed, gather_count ] => output from the scatter subworkflow, if you didn't use this subworkflow you can just use `[]` as bed since it isn't used
+    common_meta         // string:  The name of the meta field that should become the new id
```

Comment:
Most pipelines store a `sample` field for example that already contains the desired ID, I thought this would be the best way to implement it into the pipelines

Reply:
I meant most pipelines I had a look at :p (which aren't a lot so I should have maybe worded my approach differently). But I made this part optional now, so this conversion can be skipped